docs/hub/models-interacting.md

```diff
@@ -0,0 +1,159 @@
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/commit-file-dark.png"/>
+</div>
+
+3. Afterwards, click **Commit changes** to upload your model to the Hub!
+
+4. Inspect files and history
+
+You can check your repository with all the recently added files!
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/repo_with_files.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/repo_with_files-dark.png"/>
+</div>
+
+The UI allows you to explore the model files and commits and to see the diff introduced by each commit:
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/explore_history.gif"/>
+</div>
+
+5. Add metadata
+
+You can add metadata to your model card. You can specify:
+* the type of task this model is for, enabling widgets and the Inference API.
+* the used library (`transformers`, `spaCy`, etc.)
+* the language
+* the dataset
+* metrics
+* license
+* a lot more!
+
+Read more about model tags [here](/docs/hub/model-repos#model-card-metadata).
+
+6. Add TensorBoard traces
+
+Any repository that contains TensorBoard traces (filenames that contain `tfevents`) is categorized with the [`TensorBoard` tag](https://huggingface.co/models?filter=tensorboard). As a convention, we suggest that you save traces under the `runs/` subfolder. The "Training metrics" tab then makes it easy to review charts of the logged variables, like the loss or the accuracy.
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/tensorboard.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/tensorboard-dark.png"/>
+</div>
+
+Models trained with 🤗 Transformers will generate [TensorBoard traces](https://huggingface.co/transformers/main_classes/callback.html?highlight=tensorboard#transformers.integrations.TensorBoardCallback) by default if [`tensorboard`](https://pypi.org/project/tensorboard/) is installed.
+
+
+### Using Git
+
+Since model repos are just Git repositories, you can use Git to push your model files to the Hub. Follow the guide on [Getting Started with Repositories](repositories-getting-started.md) to learn about using the `git` CLI to commit and push your models.
+
+
+### Using the `huggingface_hub` client library
+
+The rich feature set in the `huggingface_hub` library allows you to manage repositories, including creating repos and uploading models to the Model Hub. Visit [the client library's documentation](https://huggingface.co/docs/huggingface_hub/index) to learn more.
+
+
+## FAQ
```

Comment:
I think creating a separate page for a FAQ would be better since there is already quite a bit of content on that page, and we want to keep the focus on interacting with models on the Hub.